.yamato/code-coverage.yml

```diff
@@ -1,16 +1,16 @@
 {% metadata_file .yamato/project.metafile %}
 ---
 {% for project in projects -%}
-code_coverage_win_{{ project.name }}:
-  name: Code Coverage Report - Windows - {{ project.name }}
+code_coverage_win_{{ project.name }}_{{ validation_editor }}:
+  name: Code Coverage Report - Windows - {{ project.name }} - {{ validation_editor }}
   agent:
     type: Unity::VM
     image: package-ci/win10:stable
     flavor: b1.large
   commands:
     - pip install unity-downloader-cli --upgrade --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-    - unity-downloader-cli -u trunk -c editor --wait --fast
+    - unity-downloader-cli -u {{ validation_editor }} -c editor --wait --fast
```

Comment:
where did you set `validation_editor` variable?
if only yamato ui, I'd suggest adding this variable to `project.metafile` as a fallback when yamato variable is not set.